perennialAutodidact/memory-snap
# 49 create setup form steps and routes

﻿## Description of changes 
I created nested routes within the setup route that will display the form component for each step. Right now there is a dummy h1 component displayed at each nested route. 


## Test steps
In the game context, change the stage value to GAME_STAGES.SETUP.  Then change the step value to ONE TWO THREE and FOUR, checking that the dummy component is displayed at each step.
